src/custom-element.js

```diff
@@ -299,6 +299,9 @@ function isInternalOrServiceNode(node) {
   if (isInternalElement(node)) {
     return true;
   }
+  if (node.tagName.indexOf('AMP-BIND') == 0) {
```

Comment:
`tagName` may be undefined if it's not an `Element` (note that `node` is a `Node`).